.pylintrc-mandatory

```diff
@@ -0,0 +1,63 @@
+[MASTER]
+load-plugins=pylint_odoo
+score=n
+
+[ODOOLINT]
+readme_template_url="https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst"
+manifest_required_authors=Odoo Community Association (OCA)
```

Comment:
```suggestion
manifest_required_authors=
```

Reply:
this conf has been taken from odoo-shopinvader. The check on the Author is disabled anyway.

Reply:
@HviorForgeFlow can you unlock it then? :)